### ISSUE TITLE
Feature request: Customizable export padding

### ISSUE THREAD

**Issue by dimateos** [excalidraw/excalidraw#6276]
I would be nice to add an slider to the export options to edit the amount of padding.

In my case, I usually use Excalidraw for reference images but when I want to copy a single one again from it it always come with a bit of padding. This behavior comes from the regular export of any kind of selected elements:

![image](https://user-images.githubusercontent.com/26868611/220968603-6115f72f-d198-464c-96bb-cb71109dcc46.png)

Also requested in #3944 for the contrary reason, adding padding.
As noted in #5380, there could also be a feature that simply puts a copy of the source image when a single one is selected.

But anyway custom padding could solve it too. Thanks for the great app!

**Comment by dwelle** [comment]
Yes, we may add support for removing padding during export. I've actually started a PR on this some time ago, but it needs some final touches. https://github.com/excalidraw/excalidraw/pull/5825
